services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,47 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  if (!event) {
+    return ERR_CODE_INVALID_QUEUE_MSG;
```

Comment:
INVALID_ARG makes more sense, INVALID_QUEUE_MSG is more for received messages from the queue that are invalid